specification/search/data-plane/Azure.Search/preview/2024-03-01-Preview/searchindex.json

```diff
@@ -1868,6 +1868,11 @@
         "exhaustive": {
           "type": "boolean",
           "description": "When true, triggers an exhaustive k-nearest neighbor search across all vectors within the vector index. Useful for scenarios where exact matches are critical, such as determining ground truth values."
+        },
+        "oversampling": {
```

Comment:
Within this domain, is this really treated as a single word, or should it be `overSampling`? I defer to your team's expertise, of course. Just wanted to raise the question.

Reply:
thanks for the callout. I double checked, it is one term in the industry. Originated in signal processing.

![image](https://github.com/Azure/azure-rest-api-specs/assets/32447910/2e8a8e51-98b1-47ca-91d1-3f51c04c3be3)